### ISSUE TITLE
Unable to install via Homebrew after 1.1.3

### ISSUE THREAD

**Issue by aydrian** [twitchdev/twitch-cli#125]
Tried to upgrade to 1.1.3 using brew and received the following.
```
==> Upgrading twitchdev/twitch/twitch-cli
  1.1.1 -> 1.1.3

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 13.1.

Error: Empty installation
```
I tried updating my command line tools. Now the CLI is gone from my system.

**Comment by aydrian** [comment]
I was able to relink version 1.1.1 but I can't upgrade to 1.1.3.

**Comment by lleadbet** [comment]
After updating the tools, what does the error read as when trying to install via `brew install`?

**Comment by aydrian** [comment]
Okay. I'm all updated. It now gives me this.
```
==> Upgrading twitchdev/twitch/twitch-cli
  1.1.1 -> 1.1.3

Error: Empty installation
```
Not sure if it matters, but I'm on an M1 MacBook Pro.

**Comment by alexweininger** [comment]
Same thing here, also on an Arm based MacBook Pro. Like mentioned above, not sure that matters though.

**Comment by lleadbet** [comment]
Have a PR to address this coming down the pipe; hope to be addressed soon.

**Comment by lleadbet** [comment]
This should be resolved with 1.1.5. Sorry about that.
